src/pages/AdminPage/components.tsx

```diff
@@ -832,23 +890,33 @@ export const ContributorsTable = ({
                       key={`user-name-${u.id}`}
                       area="wide"
                       align="left"
+                      disabled={u.deleted_at !== null}
                     >
                       <UserName user={u} />
                     </Table.Cell>
 
-                    <Table.Cell key={`address-${u.id}`}>
+                    <Table.Cell
+                      key={`address-${u.id}`}
```

Comment:
i overlooked this in the previous review... all these `key` properties on Cell are unnecessary. Only the top-level `Row` tag needs one